### PR TITLE
Update samples location / resize CodeSpaces image

### PR DESCRIPTION
**Pull request by wduraes**
With the new Arduino Repo, the samples - being all Arduino based - should be moved there to avoid duplication. 

Also, resized the   CodeSpaces printscreen as it was too large.

### CODE REVIEW

File: sdk/samples/iot/README.md [hunk 1]
```diff
@@ -69,8 +69,10 @@ To view scenario-focused examples using the API calls, please view the Azure IoT
 You can use Github Codespaces to be up and running quickly! Here are the steps to follow (assuming you already have an IoT Hub set up).
 
 1. Select the "Open with Codespaces" prompt on Github and then "New codespace".
-    ![codespace](./docs/img/codespace.png)
-1. Once the Codespace is open, all required build tools, extensions, and debugging tools will be setup for you.
+
+<img src="./docs/img/codespace.png" width="80%">
+
+2. Once the Codespace is open, all required build tools, extensions, and debugging tools will be setup for you.
```
Comment (danewalton) [inline sdk/samples/iot/README.md:75]: nit: this should be a 1. and MD automatically takes care of it

**Review by danewalton — APPROVED** [review]

**Review by ericwol-msft — APPROVED** [review]

**Comment by danewalton** [comment]
@danieljurek Might be a bug in the verify links PowerShell script.

Usually, I believe we have substitution which takes links with "main" and substitutes in the commit of the PR to check "hey, if this PR is merged to main, will the *main* link be valid in the future." Here, the verify links script is substituting the SHA of this PR's latest commit in the link to the `azure-sdk-for-c-arduino` repo, thinking it's the `azure-sdk-for-c` repo.

I think there's probably some check in the script for a "starts with this repo's url" to do the substitution for links to this repo, but it's a false positive because there's the `-arduino` suffix.

https://github.com/Azure/azure-sdk-for-c/blob/2715ed6a7ab067326ba12bde45482757e0ca3ab1/eng/common/scripts/Verify-Links.ps1#L447

Maybe here ^

Any ideas here?

**Review by danewalton — CHANGES_REQUESTED** [review]
revert verify links before merging. testing for now

**Review by danewalton — APPROVED** [review]
reverted tested changes. back to original state. added a PR to update link checker #2080

File: sdk/samples/iot/README.md (hunk 1 above)
Comment (ahsonkhan) [reply sdk/samples/iot/README.md:75]:
```suggestion
1. Once the Codespace is open, all required build tools, extensions, and debugging tools will be setup for you.
```

**Pull request merged**